src/libYARP_cv/src/yarp/cv/Cv.h

```diff
@@ -14,6 +14,9 @@
 #include <type_traits>
 #include <opencv2/core/core.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
+#if not defined(YARP_HAS_OPENCV2)
```

Comment:
Can't we just change this to: 
```suggestion
#if CV_MAJOR_VERSION >= 3
```
so we don't need to make sure that `YARP_HAS_OPENCV2` is correctly propagated to compilation units in downstream projects that use `libYARP_cv` ?

Reply:
For me it is ok :+1: